src/components/span/Span.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentProps } from "react";
+
+
+export type TSpan = {
+  classType: "stitle" | "sprice";
+} & ComponentProps<"span">;
+
+
+const joiner = (...classes: any) => classes.filter(Boolean).join(" ");
```

Comment:
We need to add this utils function in seperate folder to import any where we need